### ISSUE TITLE
Instant crash on oneplusone

### ISSUE THREAD

**Issue by tjerkw** [jariz/Dumpert#4]
Open a video -> crash. Will add stacktrace later.

**Comment by jariz** [comment]
I'm a aware of this issue, unfortunately have not had time to check it out. Appears to fail to get the proper url for the comments.  
Stacktrace:  
```
java.lang.IllegalStateException: Target host must not be null, or set in parameters. scheme=null, host=www.geenstijl.nl, path=/modlinks/
	at org.apache.http.impl.client.DefaultRequestDirector.determineRoute(DefaultRequestDirector.java:2583)
	at org.apache.http.impl.client.DefaultRequestDirector.executeOriginal(DefaultRequestDirector.java:1245)
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:702)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:694)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:793)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:767)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:756)
	at io.jari.dumpert.api.API.getComments(API.java:173)
	at io.jari.dumpert.activities.ViewItemActivity$9.run(ViewItemActivity.java:386)
	at java.lang.Thread.run(Thread.java:818)
```

**Comment by GijsTimmers** [comment]
Happened to me too. Using a MT6592 device. I see that a fix has been committed; is there any way to upgrade?

**Comment by jariz** [comment]
Hasn't been fixed. Still needs one.

**Comment by CytoDev** [comment]
@jariz, I believe this one can be closed.

**Comment by jariz** [comment]
Fixed in 1.1.4
